tests/integration/test_norm.py

```diff
@@ -76,16 +94,63 @@ def test_axis_1d(ndim, ord, keepdims):
 @pytest.mark.parametrize("ndim", range(2, LEGATE_MAX_DIM + 1))
 @pytest.mark.parametrize("ord", MATRIX_ORDS)
 @pytest.mark.parametrize("keepdims", [False, True])
-def test_axis_2d(ndim, ord, keepdims):
+@pytest.mark.parametrize(
+    "axis",
+    ((0, 1), pytest.param((1, 0), marks=pytest.mark.xfail)),
+    ids=lambda axis: f"(axis={axis})",
+)
+def test_axis_2d(ndim, ord, keepdims, axis):
+    # For all cases when axis is (1, 0) and ord is None or fro,
+    # output values of cuNumeric and Numpy are different and not close enough
     np_res = np.linalg.norm(
-        np_arrays[ndim], ord=ord, axis=(0, 1), keepdims=keepdims
+        np_arrays[ndim], ord=ord, axis=axis, keepdims=keepdims
     )
     num_res = num.linalg.norm(
-        num_arrays[ndim], ord=ord, axis=(0, 1), keepdims=keepdims
+        num_arrays[ndim], ord=ord, axis=axis, keepdims=keepdims
     )
     assert allclose(np_res, num_res)
 
 
+class TestNormErrors:
+    def test_axis_invalid_type(self):
+        # In cuNumeric, raises error in normalize_axis_tuple
+        x = num.array([1, 2, 3])
+        axis = "string"
+        with pytest.raises(TypeError):
+            num.linalg.norm(x, axis=axis)
+
+    @pytest.mark.parametrize(
+        "axis",
+        (3, -4, (1, 1), (1, 3), (1, 0, 2)),
+        ids=lambda axis: f"(axis={axis})",
+    )
+    def test_axis_invalid_value(self, axis):
+        # for (1, 1), In cuNumeric, raises error in normalize_axis_tuple
+        ndim = 2
+        with pytest.raises(ValueError):
+            num.linalg.norm(num_arrays[ndim], axis=axis)
+
+    @pytest.mark.parametrize(
+        "ndim_axis",
+        ((1, None), (2, 0)),
+        ids=lambda ndim_axis: f"(ndim_axis={ndim_axis})",
+    )
+    def test_invalid_ord_for_vector(self, ndim_axis):
+        ndim, axis = ndim_axis
+        ord = "fro"
+        msg = "Invalid norm order"
+        with pytest.raises(ValueError, match=msg):
+            num.linalg.norm(num_arrays[ndim], ord=ord, axis=axis)
+
+    def test_invalid_ord_for_matrices(self):
+        ndim = 2
+        axis = (0, 1)
+        ord = "unknown"
+        msg = "Invalid norm order for matrices"
```

Comment:
per @manopapad we decided not to check exception messages, only exception types

Reply:
Fixed.